styles.css

```diff
@@ -0,0 +1,37 @@
+.fas {
```

Comment:
If this is a library class, you should create your own class identifier instead of augmenting third-party classes.